server/app/services/mongo_pubsub.rb

```diff
@@ -93,7 +101,7 @@ def unsubscribe(subscription)
   def publish(channel, data)
     self.collection.insert_one(
       channel: channel,
-      data: data,
+      data: BSON::Binary.new(Marshal::dump(data)),
```

Comment:
But they are stored properly in mongo:
```
2.3.3 :007 > PubsubChannel.create(channel: 'foobar', data: {foo: 'bar', arr: [:foo, :bar], inner_hash: {other: :other}})
 => #<PubsubChannel _id: 591ee1e12fab7461994e44e0, created_at: nil, channel: "foobar", data: {:foo=>"bar", :arr=>[:foo, :bar], :inner_hash=>{:other=>:other}}> 
2.3.3 :008 > PubsubChannel.find('591ee1e12fab7461994e44e0')
 => #<PubsubChannel _id: 591ee1e12fab7461994e44e0, created_at: nil, channel: "foobar", data: {"foo"=>"bar", "arr"=>[:foo, :bar], "inner_hash"=>{"other"=>:other}}>
```

Just worried if this change would create any nasty surprises :/